packages/functional-tests/tests/syncV3/fxDesktopV3ForceAuth.spec.ts

```diff
@@ -166,7 +166,7 @@ test.describe('severity-1 #smoke', () => {
       const emailInputValue = await login.getEmailInput();
       expect(emailInputValue).toBe(email);
       const emailInput = await login.getEmailInputElement();
-      expect(emailInput.isDisabled());
+      expect(emailInput.isDisabled()).toBe(true);
```

Comment:
Interesting! This might be failing, but I'm not immediately sure whyyy:
```
Expected: true
Received: {}
❌ tests/syncV3/fxDesktopV3ForceAuth.spec.ts: sync v3 with an unregistered email, unregistered uid
Error: expect(received).toBe(expected) // Object.is equality
```

Reply:
same thing as above.